ansible/ansible-variables-kubeadm.yml

```diff
@@ -18,9 +18,7 @@ docker: True
 ####################################################################################
 
 ## Disable to prevent installation of Kubeadm
-kubeadm: True
-
-icp: False
+install_kubeadm: True
```

Comment:
the convention has been to just use the name of the thing being installed, e.g. "lm", "docker", "kubeadm" - any reason we need to change this as otherwise we should update all of them? Maybe use a different name for the internal one variable that tells us which platform we are installing to.

Reply:
The "kubeadm" variable indicates whether it is an installation _to_ kubeadm (similarly for "icp" variable, indicating an installation to ICP), whereas the "install_kubeadm" variable controls whether kubeadm is actually installed. There is no icp equivalent because it is always assumed that ICP has already been installed by some other mechanism. Even though we have an installation script per K8s type (kubeadm vs icp), we need the "kubeadm" and "icp" variables to distinguish between the type of install in common Ansible scripts.

Reply:
Yeah, sorry what I mean is that for all other components the property that defines if something is installed or not has a naming convention of just the component name, so it'd be more consistent to keep the control for that to be "kubeadm". 
Then the variable we use in common Ansible to distinguish between type of install could be something else, like "kubernetes_platform"? 
It's just to keep the other variables consistent.

Reply:
Right, ok. Got you. Misunderstood what you were saying.